src/main/java/com/google/opensesame/servlets/ProjectPreviewsServlet.java

```diff
@@ -21,10 +20,9 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query projectsQuery =
-        new Query("Project").addSort(ProjectEntity.NUM_MENTORS_KEY, SortDirection.DESCENDING);
+    List<ProjectEntity> projectEntities =
+        ofy().load().type(ProjectEntity.class).order("-numMentors").list();
```

Comment:
I think it would make sense to make strings into constants. So stuff like ```"numMentors"``` if it is used in multiple places. This makes changing the actual string easier in the event that we need to change it. I see that it isn't used anywhere else so this should be okay, just something to keep in mind.

Reply:
I agree with you, although keep in mind that this string corresponds with the field name in the `ProjectEntity` class, so making a constant in this case might just be moving the problem somewhere else.